lighthouse-extension/gulpfile.js

```diff
@@ -128,7 +128,13 @@ gulp.task('browserify-lighthouse', () => {
       .ignore('raven')
       .ignore('mkdirp')
       .ignore('rimraf')
-      .ignore('pako/lib/zlib/inflate.js');
+      .ignore('pako/lib/zlib/inflate.js')
+      .ignore('file')
+      .ignore('system');
+
+      // there is no way to add './doug-json-parse' to ignored packages via public API
+      // w/o browserify resolving the path into an absolute path
+      bundle._ignore.push('./doug-json-parse');
```

Comment:
I haven't figured out how to work around it using public API. `.ignore('./doug-json-parse')` will be resolved to an absolute path before being added to the blocklist, while browserify `resolve` function requires paths to match exactly.

Reply:
hmmm, that's too bad :/ but I don't think this is a huge deal personally as long as it's commented as such :)

we could maybe `require(require.resolve('./doug-json-parse'))`? but that seems bad just to please browserify